edifikana/front-end/shared-app/src/commonMain/kotlin/com/cramsan/edifikana/client/lib/features/management/hub/HubViewModel.kt

```diff
@@ -53,9 +55,12 @@ class HubViewModel(
             val organizations = organizationManager.getOrganizations().requireSuccess()
             val organizationsUIModels = organizations.map { it.toUIModel() }
 
+            val properties = propertyManager.getPropertyList().requireSuccess()
+
             updateUiState {
                 it.copy(
                     availableOrganizations = organizationsUIModels,
+                    isEmployeeTabEnabled = properties.isNotEmpty(),
```

Comment:
We should definitely still see Employees if the properties list is empty, unless we are trying to show the list of employees for the selected property. But our UI doesn't appear to be working like that